cfme/tests/cloud/test_providers.py

```diff
@@ -1486,3 +1497,50 @@ def test_add_second_provider(setup_provider, provider, request):
     second_provider.refresh_provider_relationships()
     second_provider.validate_stats(ui=True)
     assert provider.exists and second_provider.exists
+
+
+@test_requirements.cloud
+@pytest.mark.meta(automates=[1632750], blockers=[BZ(1632750,
+                                                    unblock=lambda child_provider: "object_managers"
+                                                    in child_provider)])
+@pytest.mark.uncollectif(lambda child_provider, provider:
+                        (provider.one_of(EC2Provider) and "object_managers" in child_provider) or
+                        (provider.one_of(AzureProvider) and ("object_managers" or "block_managers"
+                                                             in child_provider)),
+                        reason="Storage is not supported by AzureProvider "
+                               "and Object Storage is not supported by EC2Provider")
+@pytest.mark.provider([AzureProvider, EC2Provider, OpenStackProvider], scope="function")
+def test_cloud_provider_dashboard_after_child_provider_remove(
+        appliance, provider, request, setup_provider_funcscope, child_provider):
+    """
+        Bugzilla: 1632750
+
+        Polarion:
+        assignee: mmojzis
+        casecomponent: Cloud
+        initialEstimate: 1/6h
+        caseimportance: high
+        casecomponent: Cloud
+        testSteps:
+            1. Have a cloud provider added
+            2. Delete one of its child managers
+            3. Go to cloud provider Dashboard
+        expectedResults:
+            1.
+            2.
+            3. Dashboard should load without any issues
+    """
+    child_provider.delete()
+
+    @request.addfinalizer
+    def _wait_for_delete_provider():
+        provider.delete()
+        provider.wait_for_delete()
+
+    view = navigate_to(provider, "Details")
+    view.toolbar.view_selector.select('Dashboard View')
+    try:
+        view.flash.wait_displayed(timeout=10)
+    except Exception:
```

Comment:
I assume you want to wait for some flash message to appear and when there is some it need to be not an error, right? I assume it is not possible to do `view.wait_displayed` or equivalent. Then I think it would be good to explain this in some comment.

 Please be more specific in the `expect` phrase: `cfme.utils.wait.TimedOutError` would do.

Thanks

Reply:
I thought that `view.wait_displayed` won't work, but it actually works so I don't have to use this try/except clause.